src/yki/handler/exam_date.clj

```diff
@@ -19,13 +18,10 @@
   (fn [_oid]
     (context "/" []
       (GET "/" []
-        :query-params [{from :- ::ys/date-type nil} {days :- ::ys/days nil}]
```

Comment:
Tää query parametri taitaa itse asiassa liittyä virkailijana tutkintopäivien hakuun ja frontend antaa tässä aina arvon 180. `yki_handler/exam_session.clj`:ssä on myös tää `days` parametri käytössä siinä merkityksessä kuin se tuossa frontendin pullarissa poistettiin.

Reply:
Oho, hups joo. Näin sitä käy kun ei testaa muutoksiaan tarpeeks. Tarkoitus toki oli tuolta `yki.handler.exam-session` puolelta poistaa tämä. Toki samassa määrin turha parametri tämäkin on mielestäni, mutta ehkä tämän handlerin fiksaus voi jäädä odottamaan oikeaa tarvetta.

Reply:
Korjattu.